### PR TITLE
Director new

### PR DESCRIPTION
**Pull request by TimYuenior**
Add a new version of the director live view in line with the vision of PolyCast.

### CODE REVIEW

File: src/main/java/nl/tudelft/contextproject/gui/DirectorLiveController.java [hunk 1]
```diff
@@ -199,230 +120,236 @@ private void bindImageToBox(ImageView imgView, VBox box) {
...
+    /**
+     * Initialize the button that handles going live.
+     */
+    private void initializeLive() {
+        btnNext.setOnAction(event -> {
+            if (!endReached()) {
                 script.next(automaticCheck.isSelected());
-                updateTables();
-                nextViews();
+                // TODO: Move the current shot highlight in the table.
```
Comment (stevenmeijer) [inline src/main/java/nl/tudelft/contextproject/gui/DirectorLiveController.java]: What's with the TODO? Will that be implemented?

**Comment by stevenmeijer** [comment]
Looks good, can be merged

**Comment by bvobart** [comment]
I see two final CheckStyle warnings have been resolved. Your code looks good, nice work on the new Director interface! Once the build passes, I'll merge this.

**Comment by stevenmeijer** [comment]
Will merge this since Travis passes. Good work!

**Pull request merged**